prime-router/docs/design/hl7-validation/design.md

```diff
@@ -0,0 +1,115 @@
+# Problem Statement
+As a sender/receiver I want submitted/received data to be validated to ensure data integrity and quality.
```

Comment:
I would split this line into two so it is easier to read and more explicit/clear that there are TWO requirements. Yes I know what I did :)